### PR TITLE
Group the entries in the ChangeLog by language, not function

### PR DESCRIPTION
**Pull request by tronical**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,54 +3,60 @@ All notable changes to this project are documented in this file.
...
+ - Software renderer: Added support for linear-gradients.
+ - Software renderer: Fixed artifacts when components are deleted or elements become invisible.
+ - Fixed Infinite loop in the accessibility backend on Windows (#2195).
+ - Skia renderer: Enable anti-aliasing for rectangles with border radius.
+ - Fluent style: Adjust disabled scrollbar background color.
+ - Fixed several panics in the compiler (#2312, #2274, #2319).
+ - Winit backend: Fix rendering when moving windows between monitors with a different scale factor (#2282).
+
+
+
```
Comment (ogoffart) [inline CHANGELOG.md:18]: Extra lines ?

```suggestion

```

**Review by ogoffart — APPROVED** [review]
Makes sense!

Maybe we should have a **Breaking** tag next to breaking change?  Or is that not needed?

**Comment by tronical** [comment]
> Maybe we should have a **Breaking** tag next to breaking change? Or is that not needed?

Good idea, I'll add that.

File: CHANGELOG.md (hunk 1 above)
Comment (tronical) [reply CHANGELOG.md:18]: Ooops

**Pull request merged**